{% set name = "ftfy" %}
{% set version = "5.1.1" %}
{% set sha256sum = "67a29a2fad5f72aec2d8a0a7084e4f499ed040455133ee96b1c458609fc29e78" %}

package:
  name: {{ name }}
  version: {{ version }}

source:
  fn: {{ name }}-{{ version }}.tar.gz
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: {{ sha256sum }}

build:
  number: 0
  script: python setup.py install --single-version-externally-managed --record=record.txt
  skip: True  # [py2k]
  entry_points:
    - ftfy = ftfy.cli:main

requirements:
  build:
    - python
    - setuptools

  run:
    - python
    # Required for loading package data via `pkg_resources` module.
    - setuptools
    - wcwidth

test:
  imports:
    - ftfy
    - ftfy.bad_codecs
  requires:
    - nose
  source_files:
    - tests
  commands:
    - nosetests -v tests  # [unix]
    # CLI tests fail on windows.
    - nosetests -v -e test_cli tests  # [win]
    - ftfy --help

about:
  home: https://github.com/LuminosoInsight/python-ftfy/
  license: MIT
  license_file: LICENSE.txt
  summary: Fixes some problems with Unicode text after the fact
  dev_url: https://github.com/LuminosoInsight/python-ftfy
  doc_url: http://ftfy.readthedocs.io/

extra:
  recipe-maintainers:
    - rmax
